kolibri/deployment/default/settings/base.py

```diff
@@ -86,6 +87,10 @@
     }
 }
 
+# DIR for storing contentDBs
+CONTENT_DB_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_db"
+# DIR for storing content copies for all channels
+CONTENT_COPY_DIR = os.path.abspath(os.path.join(__file__, "../../../..")) + "/content/content_copy"
```

Comment:
Please read old issue about app based settings: https://github.com/learningequality/ka-lite/issues/4054

Since both CONTENT_COPY_DIR and CONTENT_DB_DIR are supposedly user data directories, this pattern should be relying on KOLIBRI_HOME, but should not be set in the project's settings.